examples/text_to_image/train_text_to_image.py

```diff
@@ -498,6 +536,8 @@ def deepspeed_zero_init_disabled_context_manager():
         args.pretrained_model_name_or_path, subfolder="unet", revision=args.non_ema_revision
     )
 
+    training_state = TrainingState()
```

Comment:
I'm not too happy with introducing a new concept here. Now the user needs to know both what an `accelerator` object is and what a `training_state` is.